nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -84,7 +84,7 @@ contract StakingEscrow is Issuer {
         */
         uint16 confirmedPeriod1;
         uint16 confirmedPeriod2;
-        bool reStake;
+        bool disableReStake;
```

Comment:
I personally find it easier to reason about `reStake` than `disableReStake`. disableReStake sounds like a function.

Why change the name of the variable? Why not just change reStake's default value from False to True?

Reply:
it will increase cost of first deposit plus need proper place to set default True.
False by default makes it much easier.
maybe choose other name?

Reply:
if it's not an option - then I prefer to not change anything and use second approach from #1349

Reply:
I see... maybe just change it to `reStakeDisabled` or `reStakeOff` or something?

Reply:
✔️